File: JINSEOP/프로그래머스_WEEK12/행렬의 덧셈.cpp
#include <string>
#include <vector>
using namespace std;

vector<vector<int>> add_matrix(vector<vector<int>> arr1, vector<vector<int>> arr2) {
    vector<vector<int>> answer;

    for (int i = 0; i < arr1.size(); i++) {
        vector<int> temp;
        for (int j = 0; j < arr1[i].size(); j++)
            temp.push_back(arr1[i][j] + arr2[i][j]);

        answer.push_back(temp);
    }

    return answer;
}

vector<vector<int>> solution(vector<vector<int>> arr1, vector<vector<int>> arr2) {
    vector<vector<int>> answer;

    answer = add_matrix(arr1, arr2);
    return answer;
}

File: DONGUK/프로그래머스_WEEK_12/행렬의 덧셈.py
def solution(arr1, arr2):
    answer = []
    
    for i in range(len(arr1)):
        row1, row2 = arr1[i], arr2[i]
        
        tmp = []
        for j in range(len(row1)):
            tmp.append(row1[j] + row2[j])
        
        answer.append(tmp)
        
    return answer